sql/src/main/java/io/crate/analyze/OrderBy.java

```diff
@@ -53,6 +69,55 @@ public OrderBy(List<Symbol> orderBySymbols, boolean[] reverseFlags, Boolean[] nu
         this.nullsFirst = nullsFirst;
     }
 
+    /**
+     * Creates a new OrderBy with the other symbols prepended (or returns this if symbols are empty).
+     * Symbols are de-duplicated to some degree, e.g.
+     *
+     * <pre>
+     *     ORDER BY x y `prependUnique` x z will result in x z y; without duplicating x
+     * </pre>
+     *
+     * The defaults for reverseFlags and nullsFirst are used (asc, undefined)
+     */
+    public OrderBy prependUnique(Collection<? extends Symbol> symbols) {
```

Comment:
Maybe `prependNonDuplicates` would be clearer. Thoughts?

Reply:
imho, `prependUnique` is ok, clear and concise.